dts/arm/st/wb/stm32wb.dtsi

```diff
@@ -47,6 +47,7 @@
 			label = "FLASH_CTRL";
 			reg = <0x58004000 0x400>;
 			interrupts = <4 0>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB1 0x02000000>;
```

Comment:
Can you check please, in the Ref Manual RM0434 of the stm32wb55, I see **RCC_AHB3ENR**  bit 25 for the FLASHEN

Reply:
@FRASTM yes you are right, i have put the right offset (bit 25) but not the right register.